scipy/signal/filter_design.py

```diff
@@ -299,12 +304,15 @@ def freqz(b, a=1, worN=None, whole=False, plot=None):
         A callable that takes two arguments. If given, the return parameters
         `w` and `h` are passed to plot. Useful for plotting the frequency
         response inside `freqz`.
+    fs : float
+        The sampling frequency of the digital system.
+        .. versionadded:: 1.1.0
```

Comment:
Need an extra newline before the `..` directive otherwise it doesn't render properly (not seen as separate paragraph)

https://5706-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.signal.freqz.html#scipy.signal.freqz